include/psp2/registrymgr.h

```diff
@@ -82,6 +82,99 @@ int sceRegMgrSetKeyInt(const char* category, const char* name, int buf);
  */
 int sceRegMgrSetKeyStr(const char* category, const char* name, char* buf, const int size);
 
+/**
+ * Get all keys' initial information by category (from os0:kd/registry.db0)
+ *
+ * @param category - The path to the directory to be opened (e.g. /CONFIG/SYSTEM)
+ * @param buf - Pointer to a char buffer to hold the values
+ * @param elements_number - The number of elements of the buffer
+ *
+ * @return 0 on success, < 0 on error
+ */
+int sceRegMgrGetInitVals(const char* category, char* buffer, const int elements_number);
```

Comment:
please match document's param name and function argument

Reply:
yeah